agent/CountryGetter.py

```diff
@@ -0,0 +1,17 @@
+from geoip import geolite2
+import ipgetter
+
+
+class CountryGetter(object):
+    '''
+    This class enables you to get your own country code by your IP address
+    '''
+
+    @staticmethod
+    def get_country():
+        '''
+        Returns country code of the IP address that belongs to the machine this method is called on
+        '''
+        myip = ipgetter.myip()
```

Comment:
Python style dictates the use of triple " to start and end docstrings. Please refrain from using static as I think you are meaning to use classmethod here.